packages/shared/src/lib/storage.ts

```diff
@@ -0,0 +1,9 @@
+export enum StorageTopic {
+  Squad = 'squad',
+}
+
+export const generateStorageKey = (
+  topic: StorageTopic,
+  key: string,
+  identifier = 'anonymous',
+): string => `${topic}:${key}:${identifier}`;
```

Comment:
Nice do we just document this new behaviour somewhere?
I see our indexDB has 4 different variants now.

Reply:
I agree, I also found it hard to think which one to follow, so instead, I introduced this to keep it consistent in the future. Let me write this down somewhere on our Docs page.